scripts/ci/Dockerfile.release-bundle-test

```diff
@@ -0,0 +1,33 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# Note: this file is intended only for testing the kani release bundle
+
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive \
+    DEBCONF_NONINTERACTIVE_SEEN=true
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip curl ctags && \
+    curl -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# This section is roughly (+tests) what "first time setup" will do:
+RUN rustup toolchain install nightly-2022-03-23
```

Comment:
Do we need to hardcode the path here? Can we include the toolchain file in the bundle and use that instead?

Reply:
Or I guess we can just copy the file here. That would work too.

Reply:
The next followup PR (creating the `kani-verifier` crate with proxy binaries) will solve most of the hard-coding in this Dockerfile, because it will become something like `cargo-kani setup --use-local-bundle kani-etc.tar.gz` instead of doing all this manually.

Reply:
OK... we can live with this for a couple of days.